cli/src/main/java/org/jboss/as/cli/impl/CliLauncher.java

```diff
@@ -174,6 +175,12 @@ public static void main(String[] args) throws Exception {
                         argError = "'=' is missing after --timeout";
                         break;
                     }
+                } else if(arg.startsWith("--bind=") || arg.startsWith("bind=")) {
```

Comment:
Let's not support "bind=". That will be consistent with all the other supported arguments except for the controller.